docs/compilers/CSharp/Nullability Public API Design Notes.md

```diff
@@ -16,17 +16,22 @@ This includes features like intellisense.
 
 ## General Concepts
 
-Nullability information is exposed publicly as an enum, `Nullability`.
-There are 5 possible values:
-
-* `NotApplicable` - This is used when nullability information is requested about something that does not have a nullability, such as a statement, directive, or other similar constructs.
-* `NotAnalyzed` - This is used to fulfill priority #2.
-If the user requests information while asking for nullability to not be calculated, we'll use this to stand in for nullability.
-* `Unknown` - This is used for legacy C# code that does not know about nullability.
-This can appear in modern C# code that calls unannotated C#, or in certain generic variance scenarios where a common nullability cannot be inferred from the supplied arguments.
-For use in this document unknown is represented with the `~` character, but this character does not appear in source.
-* `NotNull` - This is used when we know the nullability is not null.
-* `MayBeNull` - This is used when we know the nullability may be null.
+Nullability information is exposed publicly as two enums, `NullableAnnotation` and `NullableFlowState`.
+
+`NullableAnnotation` represents lvalues. There are 4 possible values:
+
+* `Default` - This is used when nullability information is requested about something that does not have the concept of nullability, such as a statement, directive, or other similar constructs.
```

Comment:
Sounds like `None` to me.

Reply:
I don't think `NotApplicable` works, because it will also be returned when you ask for information about VB expressions, or if we ever expose a nullability-unaware semantic model. I also had similar reservations to `None` as @jasonmalinowski.

Reply:
@333fred I guess in those cases somebody getting `NotApplicable` at least won't make them go "oh, that's a strange thing to get"....right?

Reply:
Yeah, that's not a bad point, and it does work for `NullableFlowState` as well.